### PR TITLE
Fixing broken unit test for arg_builder

### PR DESCRIPTION
**Pull request by danturn**
Apologies, i hard coded the username / password i had been using, forgetting that the way others do credentials may be different!

to solve this i changed the config in this test to use its own internal PGConfig struct which has fixed values (I think this makes more sense in here as the one in main_test is for the more "integration" style tests?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,21 +126,15 @@ git clone https://github.com/smithoss/Gonymizer.git gonymizer
 ```
 
 4. Build the project
-```
-cd gonymizer/scripts
```
Comment (danturn) [inline README.md:130]: i updated this to how i had to build it, I'm not 100% sure if the docs were out of date, or my system is different (im on debian bullseye)

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 
 
------        
+-----
```
Comment (danturn) [inline README.md:7]: my editor stripped whitespace at the end of lines in this, let me know if that's an issue and i'll revert!

**Comment by junkert** [comment]
Only test I see failing now is the `ProcessDumpFile` test. It may be my laptop that is not working properly and causing the test to fail. If it is passing for you I will go ahead and merge this. Everything else looks fine.

**Comment by danturn** [comment]
> Only test I see failing now is the `ProcessDumpFile` test. It may be my laptop that is not working properly and causing the test to fail. If it is passing for you I will go ahead and merge this. Everything else looks fine. I'll try to update instructions for running tests today if I get some time.
> 
> ```
> === RUN   TestStart/ProcessDumpFile
>     require.go:765:
>         	Error Trace:	generator_test.go:70
>         	Error:      	Expected nil, but got: &fs.PathError{Op:"open", Path:"testing/output.TestDumpFile.sql", Err:0x2}
>         	Test:       	TestStart/ProcessDumpFile
> ```

yeah, all the tests pass on my machine, 

i added to readme about the env vars... and removed the extra one i added for password in main_test.go

out of interest, do you need the POSTGRES_USER variable in main_test.go if you can set PGUSER?

**Review by junkert — APPROVED** [review]

**Comment by junkert** [comment]
No worries at all, it looks like I'll have some time this weekend to look into it and make sure things work on all versions of PSQL. Handling multiple PSQL versions seems tougher than I realized :)

**Pull request merged**